fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBMetaDataStore.java

```diff
@@ -100,6 +103,29 @@ public void setDependencies(@Nonnull Descriptors.FileDescriptor[] dependencies)
         this.dependencies = dependencies;
     }
 
+    @Nullable
+    public Descriptors.FileDescriptor getLocalFileDescriptor() {
+        return localFileDescriptor;
+    }
+
+
+    /**
+     * Set local meta-data file descriptor.
+     *
+     * <p>
+     * This can optionally take a local file descriptor. A record store created off of the meta-data store may not be
```

Comment:
I think this verbiage about optionally taking a local file descriptor was taken from a previous version where this referred to using a `localFileDescriptor` as an optional parameter (such as that exists in Java). I think this paragraph works without the first sentence.